client/src/annoMatrix/views.ts

```diff
@@ -4,37 +4,59 @@
 Views on the annomatrix.  all API here is defined in viewCreators.js and annoMatrix.js.
 */
 import clip from "../util/clip";
-import AnnoMatrix from "./annoMatrix";
-import { _whereCacheCreate } from "./whereCache";
+import AnnoMatrix, { ObsColumnValue } from "./annoMatrix";
+import { _whereCacheCreate, WhereCache } from "./whereCache";
 import { _isContinuousType, _getColumnSchema } from "./schema";
+import {
+  Dataframe,
+  DenseInt32Index,
+  IdentityInt32Index,
+  KeyIndex,
+} from "../util/dataframe";
+import { Query } from "./query";
+import {
+  AnnotationColumn,
+  ColumnSchema,
+  Field,
+  LayoutColumn,
+} from "../common/types/entities";
 
-class AnnoMatrixView extends AnnoMatrix {
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any --- FIXME: disabled temporarily on migrate to TS.
-  constructor(viewOf: any, rowIndex = null) {
-    // @ts-expect-error ts-migrate(2531) FIXME: Object is possibly 'null'.
+type MapFn = (
+  field: Field,
+  colLabel: string,
+  colSchema: ColumnSchema,
+  colData: string[],
```

Comment:
typing of `colData` and return value are wrong.
* colData can be any Array or TypedArray
* return value is same type as colData param

Reply:
Updated [views.ts::_clipAnnoMatrix](https://github.com/chanzuckerberg/cellxgene/blob/mim/annoMatrix/client/src/annoMatrix/views.ts#L247).